operator-framework-quarkus-extension/deployment/src/main/java/io/javaoperatorsdk/quarkus/extension/deployment/QuarkusExtensionProcessor.java

```diff
@@ -62,6 +73,78 @@ void indexSDKDependencies(
         new IndexDependencyBuildItem("io.javaoperatorsdk", "operator-framework-core"));
   }
 
+  @BuildStep
+  void createConfigurationServiceAndOperator(
```

Comment:
Could this be done during the build step as the configuration creation step? If not, why not?

Reply:
It can't, otherwise there are cycles.